src/main/kotlin/nl/knaw/huc/broccoli/resources/projects/V1ProjectsResource.kt

```diff
@@ -0,0 +1,28 @@
+package nl.knaw.huc.broccoli.resources.projects
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.jayway.jsonpath.ParseContext
+import jakarta.ws.rs.client.Client
+import nl.knaw.huc.broccoli.core.Project
+import org.slf4j.LoggerFactory
+
+class V1ProjectsResource(
+    private val projects: Map<String, Project>,
+    client: Client,
+    jsonParser: ParseContext,
+    jsonWriter: ObjectMapper
+) : ProjectsResource(
+    projects, client, jsonParser, jsonWriter
+) {
+
+    override fun listProjects(): Set<String> {
+        logger.info("List projects v1")
+        return projects.keys
+    }
+
+    companion object {
+        private val logger =
+            LoggerFactory.getLogger(V2ProjectsResource::class.java)
```

Comment:
This initialises the V1 logger with V2 class. ;-)

Reply:
Gefixt in [7e152a6](https://github.com/knaw-huc/broccoli/pull/77/commits/7e152a6764e48caf37b86351ff3d1ac209a6d2f4)